### DISCUSSION TITLE
Find URLs inside comments

### DISCUSSION THREAD

**Discussion by ghost** [lycheeverse/lychee#1358] [Ideas] [answered]
I think a really neat feature to add would be the ability to read urls that are inside comments of files outside of just HTML and Markdown but CSS, JS/TS and many more.

**Comment by mre** [comment]
Do you have an example?
This works at least:

```bash
❯❯❯ cat test.css
/* https://corrode.dev */
a {
  color: hotpink;
}
❯❯❯ lychee -vv test.css
✔ [200] https://corrode.dev/

🔍 1 Total (in 0s) ✅ 1 OK 🚫 0 Errors
```

**Reply by ghost** [reply]
My idea is that it would like `lychee ./docs` works with find links in html and markdown files but this approach works too. Does this work for other languages? I didn't find `-vv` in the [command line options](https://lychee.cli.rs/#/usage/cli).

**Reply by mre** [reply, marked as answer]
You can use globs:

```
lychee '**/*.css'
```

This would search for broken links in all CSS files in all subdirectories.

You can use multiple globs:

```
lychee '**/*.css' '**/*.html' '**/*.md' '**/*.ruby' 
```

A quick way to list all the searched inputs would be

```
lychee --dump-inputs '**/*.css' '**/*.html' '**/*.md' '**/*.ruby' 
```

You can find more info on glob patterns [here](https://en.wikipedia.org/wiki/Glob_(programming)).

The `-vv` is the same as `-v -v`, which means "extra verbose". It is a Unix convention.
